src/Uno.UI/UI/Xaml/Controls/Control/Control.cs

```diff
@@ -718,7 +718,7 @@ public Thickness Padding
 				typeof(Control),
 				new FrameworkPropertyMetadata(
 					Thickness.Empty,
-					FrameworkPropertyMetadataOptions.AffectsMeasure,
+					FrameworkPropertyMetadataOptions.None,
```

Comment:
@ramezgerges Following on your comments, why is it not needed for Control to affect measure? In practice, it would assume should not change much since this property is generally template-bound to another control and a control instance itself that not do much, but I wonder what you saw there.

Reply:
Having AffectsMeasure on Control.PaddingProperty causes derived elements to get their measure affected when they shouldn't (I tried with GridView and UserControl, I think). @MartinZikmund was actually the one to first notify me of this, before I started adding the runtime tests.